src/routers/participant.router.ts

```diff
@@ -1,60 +1,50 @@
 import { Router } from "express";
 import {
-    acceptParticipantByPhone,
-    addNoteToParticipant,
-    addParticipant,
-    bulkEmailParticipants,
-    deleteParticipantByEmail,
-    deleteParticipantByPhone,
-    emailParticipantByPhone,
-    getAllParticipants,
-    rejectParticipantByPhone,
-    updateParticipantByPhone,
-    updateParticipantStatus,
+  acceptParticipantByPhone,
+  addNoteToParticipant,
+  addParticipant,
+  bulkEmailParticipants,
+  deleteParticipantByEmail,
+  deleteParticipantByPhone,
+  emailParticipantByPhone,
+  getAllParticipants,
+  rejectParticipantByPhone,
+  updateParticipantByPhone,
+  updateParticipantStatus,
 } from "../controllers/participant.controller";
-import { mwValidateParticipant, mwValidateStatus } from "../middlewares/participants/participant.validation.middleware";
+import {
+  mwValidateParticipant,
+  mwValidateStatus,
+} from "../middlewares/participants/participant.validation.middleware";
 import isLoggedIn from "../middlewares/authentication/login.middleware";
 import giveAccessTo from "../middlewares/authentication/giveAccessTo.middleware";
+import { limiter } from "../middlewares/rate-limiter";
 
 const participantRouter = Router();
 
 participantRouter
-.get(
-    "/getAll",
-    isLoggedIn,
-    giveAccessTo("member"),
-    getAllParticipants
-)
-.post("/add", mwValidateParticipant, addParticipant)
-.patch(
-    "/update",
-    isLoggedIn,
-    giveAccessTo("hr"),
-    updateParticipantByPhone
-)
-.delete(
+  .use(limiter)
+  .get("/getAll", isLoggedIn, giveAccessTo("member"), getAllParticipants)
+  .post("/add", mwValidateParticipant, addParticipant)
```

Comment:
The `/add` endpoint in participants need a dedicated limiter to limit it stricter than the others. Could be limited to 4 submissions per 2 hours for example.